apps/web/__e2e__/models/settings-view.model.ts

```diff
@@ -59,4 +72,19 @@ export class SettingsViewModel {
   async isLoggedIn() {
     return await this.accountStatusContainer.isVisible();
   }
+
+  async createBackup(encryptBackups = false, password = "") {
```

Comment:
no need to specify 2 parameters. Just use `password` to see if we should encrypt or not.